db/migrate/20170905104611_create_ip_addresses.rb

```diff
@@ -0,0 +1,10 @@
+class CreateIpAddresses < ActiveRecord::Migration
```

Comment:
In thinking about this a bit more, I think it makes more sense to store the [IP addresses as integers](http://www.boffbowsh.co.uk/2012/05/22/serving-memcached-pages-from-nginx.html) on the `EventLog` record. We shouldn't be storing them as `VARCHAR`s anyway, so once you have an integer pointing to another table of integers we should just cut out the middleperson.

Reply:
Do we need to worry about ipv6 addresses?

Reply:
I... don't believe so. What does `remote_ip` return.

Reply:
When I was testing it locally, it could return either ipv4 and ipv6

Reply:
I don't know how long it'll be until we're supporting IPv6 in our stack. I think it makes sense to only handle the IPv4 case for now. If we get an IPv6 address, record it as a NULL and log an error to Sentry.